### DISCUSSION TITLE
Setting for setting docs on sub-path

### DISCUSSION THREAD

**Discussion by silveltman** [withastro/starlight#1447] [Feature Requests]
### What version of `starlight` are you using?

0.15.2

### What is your idea?

I would like to have the docs pages under a subpath, like `/docs`. This could be done in the config.

### Why is this feature necessary?

I use my astro repo/project for a combination of the project itself AND the docs. In one and the same repo.

In my case it's kind of a starter-project for marketing website. I want to be able to test the pages in this repo too. Right now the urls from my starter can conflict with the urls from the docs

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by silveltman** [comment]
sorry, already an request for this feature

**Reply by delucis** [reply]
Already supported too! https://starlight.astro.build/manual-setup/#use-starlight-at-a-subpath

Although there are some improvements we’d still like to work on: #966
